Hybrid/src/main/java/com/microsoft/azure/management/compute/samples/ManageVirtualMachine.java

```diff
@@ -0,0 +1,368 @@
+            DataDisk lun1Disk = new DataDisk().withLun(1).withDiskSizeGB(1).withCaching(CachingTypes.READ_ONLY)
+                    .withCreateOption(DiskCreateOptionTypes.EMPTY).withName(dataDiskName)
+                    .withVhd(new VirtualHardDisk().withUri(dataDiskVhdUri));
+            vmStorageProfile.dataDisks().add(lun1Disk);
+
+            // Create Linux Vm
+            //
+            Date t1 = new Date();
+            VirtualMachine virtualMachine = azure.virtualMachines().define(linuxVMName).withRegion(location)
+                    .withExistingResourceGroup(rgName).withHardwareProfile(vmHardwareProfile).withOsProfile(vmOsProfile)
+                    .withNetworkProfile(vmNetworkProfile).withStorageProfile(vmStorageProfile).create();
+
+            Date t2 = new Date();
+            System.out.println(
+                    "Created VM: (took " + ((t2.getTime() - t1.getTime()) / 1000) + " seconds) " + virtualMachine.id());
+                    
+            // Print virtual machine details
+            Utils.print(virtualMachine);
+
+            // ============================================================= 
+            // Update - Tag the virtual machine
+
+            virtualMachine.update().withTag("who-rocks", "java").withTag("where", "on azure").apply();
+            System.out.println("Tagged VM: " + virtualMachine.id());
+
+
+            // ============================================================= 
+            // Update - Add data disk
+            //
+            String newdataDiskName = "dataDisk2";
+            String newdataDiskVhdName = newdataDiskName + ".vhd";
+            String newdataDiskVhdUri = storageAccount.primaryEndpoints().blob() + "test" + "/" + newdataDiskVhdName;
+            DataDisk newDisk = new DataDisk().withLun(2).withDiskSizeGB(1).withCaching(CachingTypes.READ_ONLY)
+                    .withCreateOption(DiskCreateOptionTypes.EMPTY).withName(newdataDiskName)
+                    .withVhd(new VirtualHardDisk().withUri(newdataDiskVhdUri));
+            
+                    vmStorageProfile.dataDisks().add(newDisk);
+            virtualMachine.update().withStorageProfile(vmStorageProfile).apply();
+
+            System.out.println("Added a data disk to VM" + virtualMachine.id());
+            Utils.print(virtualMachine);
+
+            // ============================================================= 
+            // Update - detach data disk
+
+            vmStorageProfile.dataDisks().remove(0);
+            virtualMachine.update().withStorageProfile(vmStorageProfile).apply();
+
+            System.out.println("Detached data disk at lun 1 from VM " + virtualMachine.id());
+
+            // ============================================================= 
+            // Restart the virtual machine
+
+            System.out.println("Restarting VM: " + virtualMachine.id());
+
+            virtualMachine.manager().virtualMachines().inner().restart(rgName, linuxVMName);
```

Comment:
we should use inner only if it is necessary, this can be modified as 

```java
irtualMachine.manager().virtualMachines().restartAsync(rgName, linuxVMName).toCompletable().await();
```